ui/src/components/EventsTable/index.tsx

```diff
@@ -0,0 +1,225 @@
+  const handleNextButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => onChangePage(event, page + 1)
+  const handleLastPageButtonClick = (event: React.MouseEvent<HTMLButtonElement>) =>
+    onChangePage(event, Math.max(0, Math.ceil(count / rowsPerPage) - 1))
+
+  return (
+    <Box display="flex" className="test-box">
+      <IconButton onClick={handleFirstPageButtonClick} disabled={page === 0} aria-label="first page">
+        <FirstPageIcon />
+      </IconButton>
+      <IconButton onClick={handleBackButtonClick} disabled={page === 0} aria-label="previous page">
+        <KeyboardArrowLeftIcon />
+      </IconButton>
+      <IconButton
+        onClick={handleNextButtonClick}
+        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
+        aria-label="next page"
+      >
+        <KeyboardArrowRightIcon />
+      </IconButton>
+      <IconButton
+        onClick={handleLastPageButtonClick}
+        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
+        aria-label="last page"
+      >
+        <LastPageIcon />
+      </IconButton>
+    </Box>
+  )
+}
+
+const format = (date: string) => day(date).format('YYYY-MM-DD HH:mm:ss')
+
+interface EventsTableProps {
+  events: Event[] | null
+}
+
+const EventsTable: React.FC<EventsTableProps> = ({ events }) => {
+  const classes = useStyles()
+
+  const [order, setOrder] = useState<Order>('desc')
+  const [orderBy, setOrderBy] = useState<keyof SortedEvent>('CreatedAt')
+  const [page, setPage] = React.useState(0)
+  const [rowsPerPage, setRowsPerPage] = React.useState(5)
+
+  const handleSortEvents = (_: React.MouseEvent<unknown>, k: keyof SortedEvent) => {
+    const isAsc = orderBy === k && order === 'asc'
+
+    setOrder(isAsc ? 'desc' : 'asc')
+    setOrderBy(k)
+  }
+
+  const handleChangePage = (_: React.MouseEvent<HTMLButtonElement> | null, newPage: number) => setPage(newPage)
+
+  const handleChangeRowsPerPage = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    setRowsPerPage(parseInt(event.target.value))
+    setPage(0)
+  }
+
+  return (
+    <TableContainer className={classes.tableContainer}>
+      <Table>
+        <EventsTableHead order={order} orderBy={orderBy} onSort={handleSortEvents} />
+
+        <TableBody>
+          {events &&
+            stableSort<SortedEvent>(events, getComparator(order, orderBy))
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((e) => (
+                <TableRow key={e.ID} hover>
+                  <TableCell>{e.Experiment}</TableCell>
+                  <TableCell>{e.Kind}</TableCell>
+                  <TableCell>{e.Namespace}</TableCell>
+                  <TableCell>{format(e.CreatedAt)}</TableCell>
+                  <TableCell>{format(e.UpdatedAt)}</TableCell>
+                  <TableCell>{format(e.StartTime)}</TableCell>
+                  <TableCell>{format(e.FinishTime)}</TableCell>
```

Comment:
![Screenshot from 2020-06-15 11-39-27](https://user-images.githubusercontent.com/25150124/84677593-eed2b780-aefc-11ea-80d4-112f19cf09ed.png)

Getting invalid date here since the event is not finished. We should add a check this case

Reply:
Resolved in 5647ea8.